src/openrct2/network/NetworkBase.cpp

```diff
@@ -2693,13 +2693,13 @@ void NetworkBase::Client_Handle_MAP([[maybe_unused]] NetworkConnection& connecti
             GameLoadScripts();
             GameNotifyMapChanged();
             _serverState.tick = gCurrentTicks;
-            // window_network_status_open("Loaded new map from network");
+            // WindowNetworkStatusOpen("Loaded new map from network");
```

Comment:
Bumped into this function that has been renamed already a few months back, just not this comment 😄